modules/dreamview/conf/vehicle_data.pb.txt

```diff
@@ -169,4 +169,8 @@ data_files {
 data_files {
   source_path: "perception_conf/pointcloud_preprocessor.conf"
   dest_path: "/apollo/modules/perception/production/data/perception/lidar/models/pointcloud_preprocessor/velodyne128/pointcloud_preprocessor.conf"
-}
\ No newline at end of file
+}
+data_files {
+  source_path: "perception_camera_params"
```

Comment:
camera_params

Reply:
There is exist 'camera_params' path, but destination path is not 'perception/camera/params'.